### ISSUE TITLE
[UX and DevOps] Datadog working session

### ISSUE THREAD

**Issue by BerniXiongA6** [department-of-veterans-affairs/va.gov-cms#16541]
## Background
In order for UX to begin work on enhancing the monitoring experience to include user metrics, behaviors, etc., a working session with DevOps needs to occur so that UX can move forward on implementing Real User Monitoring.

## Acceptance Criteria
- [x] A working session is held between DevOps and UX
- [x] UX needs of Datadog have been identified

## To Do (optional for Assignee, if needed for tasking the work)

**Comment by BerniXiongA6** [comment]
Hi @BlakeOrgan @ariperez -- here's the ticket I promised to create for you both to schedule a working session for this sprint to discuss UX needs of Datadog. This is just the working session ticket. Thanks! cc: @maortiz-27-80

**Comment by BlakeOrgan** [comment]
@EWashb @maortiz-27-80 @BerniXiongA6 

Meeting was held with me, @joagnitti, and @ariperez Dec 20.

Ari set up a Datadog dashboard with what Real User Monitoring comes with out of the box and a tugboat preview that's linked to the dashboard. All that info can be found in [ticket #16526](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16526).

UX explored the baseline analytics that are collected and determined that we thought it would be a good idea to see this implemented on Prod to start collecting some baseline analytics. After we've got a baseline of data, we can start collecting more specific data as different situations arise.

A couple of things I noticed upon first inspection of the test dashboard:

- Not all sessions include a session replay. I'm not sure what determines whether one is available to watch or not.
- If you watch some of the session replays, no data actually shows up in the textboxes or dropdowns of the CMS to be able to see what info users are putting in. Surfacing that information could give us a clearer picture into the content users are putting into the CMS.
- I didn't see a way to show which user I was looking at and what user group they belong to (VAMC, Vet Center, etc). I think that will be important info to have as we're trying to parse out how different groups behave.
